css/skin/summer-orange.css

```diff
@@ -263,23 +263,22 @@ a, a:hover, a:focus {
 }
 
 #sponsors-section img{
-    opacity: 0.5;
+    transition: all .2s linear;
```

Comment:
Since you're editing the CSS file, can you change the version number for the respective CSS file in `index.html`.
So now, the link tag would be something like `<link rel="stylesheet" href="css/skin/summer-orange.css?v=1.0.1">`. This is to ensure that the browser automatically gets the corresponding new CSS file and doesn't rely on the cache.

See https://github.com/HackCU/LocalHackDay/pull/9